docs/source/configuration/volto-slate/writing-plugins.md

```diff
@@ -136,16 +143,84 @@ export const withTooltip = (editor) => {
 };
 ```
 
-### Volto configuration registry
+The constant `TOOLTIP` used throughout the plugin is defined in `editor/plugins/TooltipPlugin/constants.js`.
+
+```js
+export const TOOLTIP = 'tooltip';
+```
+
+## Volto configuration registry
 
-Finally register the plugin in Volto's configuration.
+Finally register the plugin and the toolbar button in Volto's configuration.
 
 ```js
-import installEditor from './editor';
-export default function install(config) {
-  config.settings.tooltip = [...(config.settings.tooltip || []), TOOLTIP];
-  config = installEditor(config);
+
+import installTooltipPlugin from './editor/plugins/TooltipPlugin';
+import { TOOLTIP } from './editor/plugins/TooltipPlugin/constants';
+
+const applyConfig = (config) => {
+  slate.toolbarButtons = [...(slate.toolbarButtons || []), TOOLTIP];
+  slate.expandedToolbarButtons = [
+    ...(slate.expandedToolbarButtons || []),
+    TOOLTIP,
+  ];
+  config = installTooltipPlugin(config);
```

Comment:
Thanks @ksuess . I overlooked these variable names and syntaxes when writing, Looking more readable now! 🎉